test/ui/cypress/e2e/login_with_RH_account.cy.js

```diff
@@ -0,0 +1,7 @@
+/// <reference types="cypress" />
```

Comment:
We don't need this file at all, it only runs one command however that command is already used in every single test so running it again is redundant.

Reply:
@tznamena , the reason that I created this spec file is I want to count Deployment Engine UI's login as a separate test scenario since it has some logic integrated with RH account login. What is it a good way to re-organize it?